src/react-query/queryClient.ts

```diff
@@ -1,47 +1,51 @@
 import { QueryClient } from '@tanstack/react-query';
-import AsyncStorage from '@react-native-async-storage/async-storage';
-import { createAsyncStoragePersister } from '@tanstack/query-async-storage-persister';
-import { PersistQueryClientOptions } from '@tanstack/react-query-persist-client';
-import { MMKV } from 'react-native-mmkv';
-import { queryStorage } from '@/storage/legacy';
+import { PersistedClient, Persister, PersistQueryClientOptions } from '@tanstack/react-query-persist-client';
+import { debounce } from 'lodash';
+import { REACT_QUERY_STORAGE_ID, queryStorage } from '@/storage/legacy';
+
+const ENABLE_LOGS = false;
+
+class MMKVPersister implements Persister {
+  private static readonly throttleMs = 5000;
+
+  private throttledPersist = debounce(
+    (persistedClient: PersistedClient) => {
+      if (ENABLE_LOGS) console.log('Persisting client');
```

Comment:
We should use logger.debug and we can get rid of `ENABLE_LOGS`

Reply:
Don't really need to log here, just removed this code.